modules/tinymce/src/plugins/paste/test/ts/browser/SmartPasteTest.ts

```diff
@@ -35,19 +36,53 @@ UnitTest.asynctest('browser.tinymce.plugins.paste.SmartPasteTest', (success, fai
     LegacyUnit.equal(SmartPaste.isAbsoluteUrl(''), false);
   });
 
-  suite.test('TestCase-TBA: Paste: isImageUrl', function () {
-    LegacyUnit.equal(SmartPaste.isImageUrl('http://www.site.com'), false);
-    LegacyUnit.equal(SmartPaste.isImageUrl('https://www.site.com'), false);
-    LegacyUnit.equal(SmartPaste.isImageUrl('http://www.site.com/dir-name/file.jpeg'), true);
-    LegacyUnit.equal(SmartPaste.isImageUrl('http://www.site.com/dir-name/file.jpg'), true);
-    LegacyUnit.equal(SmartPaste.isImageUrl('http://www.site.com/dir-name/file.png'), true);
-    LegacyUnit.equal(SmartPaste.isImageUrl('http://www.site.com/dir-name/file.gif'), true);
-    LegacyUnit.equal(SmartPaste.isImageUrl('https://www.site.com/dir-name/file.gif'), true);
-    LegacyUnit.equal(SmartPaste.isImageUrl('https://www.site.com/~dir-name/file.gif'), true);
-    LegacyUnit.equal(SmartPaste.isImageUrl('https://www.site.com/dir-name/file.gif?query=%42'), false);
-    LegacyUnit.equal(SmartPaste.isImageUrl('https://www.site.com/dir-name/file.html?query=%42'), false);
-    LegacyUnit.equal(SmartPaste.isImageUrl('file.gif'), false);
-    LegacyUnit.equal(SmartPaste.isImageUrl(''), false);
+  suite.test('TestCase-TBA: Paste: isImageUrl', function (editor) {
```

Comment:
This would probably be a good candidate for a property based atomic test. It would need some refactoring though to not pass an editor. and instead pass the valid extensions.

- [ ] Consider adding a property test for the isImageUrl logic

Reply:
Discussed on slack, we're not going to worry about the above, however we should add a test to ensure a custom format is transformed on paste (similar to the existing `.jpg` ones).

Reply:
Discussed in slack. Going to add one more E2E(ish) test for `image_file_type` settings